.github/workflows/ci.yml

```diff
@@ -277,7 +277,7 @@ jobs:
         cargo test --features "pg$PG_VER verify" --no-default-features --lib plrust
 
     - name: Run PL/Rust + postgrestd tests
-      if: matrix.os == 'ubuntu-20.04' && matrix.version == 'postgres-14'
+      if: matrix.version == 'postgres-14' || matrix.version == 'postgres-15' && matrix.os == 'ubuntu-20.04'
```

Comment:
For a split second I was slightly concerned about order of operations here, but then quickly realized that it's all good.

Reply:
reverted anyways.